job_board/tests/test_search_view.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+
+valid_form_data = [
+    {
+        'job_type': '1',
+        'location': '3',
+        'years_of_experience': 'no experience',
+        'work_schedule': 'Flexible'
+    },
+]
+
+invalid_form_data = [
+    {
+        'job_type': '-3',
+        'location': '2',
+        'years_of_experience': '5+ years',
+        'work_schedule': 'Not specified'
+    }
+]
+
+JOB_BOARD_URL = '/job_board/'
+
+
+@pytest.mark.django_db
+class TestSearchView():
+
+    def test_get_request(self, client):
+        response = client.get(JOB_BOARD_URL)
+        assert response.status_code == 200
+
+    def test_search_using_valid_form(self, client):
```

Comment:
You can add fixture for the form content, no need of static variables
Both tests returns 200OK how do you know pages generated created properly? 

What happens when non/successful form submission created? the difference should be tested

Post method in view returns this 
return render(request, 'job_board/job_board.html', context)

I would recommend to check the context

Reply:
@Yarboa Added the relevant fixture.
Also, I am checking a successful form submission by searching for the use of the job_board/post.html template which represents a search result.